pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -177,41 +43,11 @@ The main repositories are:
 - `memgraph/memgraph` includes Memgraph database and command-line interface
```

Comment:
Is memgraph/lab missing here?

Reply:
I was thinking about adding it. I'll add it.

Reply:
Added.